packages/deployments/contracts/src/cli/init/config/testnet/production.ts

```diff
@@ -132,6 +132,41 @@ export const TESTNET_PRODUCTION_INIT_CONFIG: InitConfig = {
         },
       },
     },
+    {
+      name: "P8",
+      canonical: {
+        domain: "6778479",
+        address: "0x772fCe4B8E88BD19e86dC92428d242704aC480a0",
+        decimals: 18,
+      },
+      representations: {
+        // mainnet
+        "6648936": {
```

Comment:
If a token is not present for any chain, you can just not add that particular domain (chain) in the config. It will work fine, No need for zero address.

Reference Link: https://github.com/connext/monorepo/blob/79dae16d9f88f19cfd8a06d3a567899fc920ed92/packages/deployments/contracts/src/cli/init/config/testnet/production.ts#L136